<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>420</width>
      <height>200</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Setup</string>
     </attribute>
     <widget class="QComboBox" name="selectedDay">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentText">
       <string>Every Sunday</string>
      </property>
      <item>
       <property name="text">
        <string notr="true">Every Sunday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Every Monday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Every Tuesday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Every Wednesday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Every Thursday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Every Friday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Every Saturday</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>344</x>
        <y>212</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>216</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>11:15:49 20.10.2019</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,4,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QRadioButton" name="specifyRadio">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Specified time</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="dailyRadio">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Daily</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="weeklyRadio">
         <property name="text">
          <string>Weekly</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>111</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="sleepRadio">
         <property name="text">
          <string>Sleep</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="wakeRadio">
         <property name="text">
          <string>Wake up</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="shutdownRadio">
         <property name="text">
          <string>Shut down</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="restartRadio">
         <property name="text">
          <string>Restart</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>173</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Swis721 BlkCn BT</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="keyboardTracking">
       <bool>true</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="date">
       <date>
        <year>2019</year>
        <month>10</month>
        <day>21</day>
       </date>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>100</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Swis721 BlkCn BT</family>
        <pointsize>20</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="time">
       <time>
        <hour>16</hour>
        <minute>42</minute>
        <second>1</second>
       </time>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>31</width>
        <height>161</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Running tasks</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>471</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>113</number>
         </attribute>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Task</string>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>When</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>300</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dailyRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dailyRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectedDay</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>weeklyRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>specifyRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectedDay</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
